android/app/build.gradle

```diff
@@ -32,7 +32,7 @@ if (keystorePropertiesFile.exists()) {
 }
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
```

Comment:
Does this remove support for 20% of Android users? https://gs.statcounter.com/os-version-market-share/android

Reply:
Ah, target vs. minimum. I need to read about the differences. Ignore me.